frontend/Nouns/CreateEditIntakeForm.tsx

```diff
@@ -164,17 +161,7 @@ export function CreateEditIntakeForm(
                           })
                         }
                       >
-                        <FormField className="pointer-events-none mt-3">
-                          <FormFieldLabel htmlFor={`intake-item-${item.id}`}>
-                            {fieldItem.question.label}
-                          </FormFieldLabel>
-                          <Input
-                            id={`intake-item-${item.id}`}
-                            placeholder={fieldItem.question.placeholderText}
-                            required={fieldItem.question.required}
-                            disabled
-                          />
-                        </FormField>
+                        <ViewIntakeItem intakeItem={item} />
```

Comment:
The code looks so much cleaner.  Nice refactor to make ViewIntakeItem it's own component.